server/src/main/java/com/objectcomputing/checkins/services/private_notes/PrivateNotesBadArgException.java

```diff
@@ -0,0 +1,7 @@
+package com.objectcomputing.checkins.services.private_notes;
```

Comment:
can we use com.objectcomputing.checkins.services.exceptions.BadArgException instead of creating a new class?

Reply:
updated to use BadArgException.